website/docs/guides/migration/from-v2-to-v3.md

```diff
@@ -0,0 +1,204 @@
+---
+sidebar_position: 1
+---
+
+# From v2 to v3
+
+KafkaFlow version 3 brings several significant changes and improvements. This guide will help you navigate through the migration process from version 2 to version 3.
+
+## Breaking Changes
+
+The update to v3 introduces some breaking changes. Please take them into consideration when updating KafkaFlow:
+
+- [UI Dashboard URL Change](#2-breaking-change---ui-dashboard-url-change-303)
+- [Removed Packages and Core Changes](#3-breaking-change---remove-packages-and-core-changes)
+
+## 1. Update to Core Libraries to .NET 6 ([#298](https://github.com/Farfetch/kafkaflow/issues/298), [#322](https://github.com/Farfetch/kafkaflow/issues/322))
```

Comment:
We can use information hierarchy. 
If the previous "Breaking Changes" section is an H2, here we should use an H3 if we want to group inside of those.

Reply:
Restructured the document to divide between "Breaking Changes" and "Feature".

Structured the headings accordingly.

Also added the table of contents since with VS Code the plugin already generated and updates the table of contents automatically.

The issue links in the headings were messing up the table os contents, so referenced them in another way. Please check.